src/models/permissions/permission-response.model.ts

```diff
@@ -0,0 +1,8 @@
+
+import { Permission } from './permission.model';
+
+export interface PermissionResponse {
+  objectName: string;
```

Comment:
how about adding here types - then we get auto complete 
 also https://github.com/coresystemsFSM/fsm-sdk/blob/master/src/model/DTOModels.ts#L1
like in the FSM sdk ?